resources/views/home/search.blade.php

```diff
@@ -68,7 +68,7 @@ class="search-header__input js-search--input"
                 @else
                     <div class="search-result">
                         <div class="search-result__row search-result__row--notice">
-                            {{ trans('home.search.missing_query', ['n' => config('osu.search.minimum_length')]) }}
+                            {{ trans('home.search.missing_query', ['n' => 1]) }}
```

Comment:
just so existing translations don't show `minimum :n characters`

Reply:
Probably worth just renaming the key to force a new translation?